lib/hotel.rb

```diff
@@ -0,0 +1,127 @@
+require 'pry'
+require_relative 'room'
+require_relative 'reservation'
+
+class Hotel
```

Comment:
One note on the name.  This class really doesn't do all the features of a Hotel.  Instead it manages Reservations.  So a more descriptive name would be appropriate.